_specs/ecip-1102.md

```diff
@@ -0,0 +1,139 @@
+---
+ecip: 1102
+title: Use The Etherplan Consensus Engine for Approving Proto Treasury Funding Proposals
+lang: EN
+author: Donald McIntyre (@TokenHash)
+discussions-to: https://github.com/ethereumclassic/ECIPs/issues/407
+status: WIP
+type: Meta
+created: 2021-02-03
+complementary to: ECIP-1098 Proto Treasury System 
+license: CC-BY-SA
+---
+
+## Abstract
+
+The Ethereum Classic (ETC) [Proto Treasury System](https://github.com/ethereumclassic/ECIPs/blob/master/_specs/ecip-1098.md) (p-ECTS) proposal suggests directing 10% of the miner tax funds to a [Gitcoin grants](https://gitcoin.co/grants/) program. This ECIP proposes to direct those funds to an ETC community managed pool on the [Etherplan Consensus Engine](https://etherplan.com/2021/02/02/the-etherplan-consensus-engine/14989/) (ECE) dapp. The ECE is a system designed for decentralized groups of any kind to distribute funds using rough consensus.
```

Comment:
If what you mean by "adapting" is to make this ECIP-1102 part of ECIP-1098, or to just add the text here to 1098 as an option, so they are not two separate ones, that would be ok. I think two separate ones is better because this ECIP proposes a somewhat complicated addition (it requires pos and pow blockchains that need to interoperate, a web interface, several smart contracts, a full evaluation and test of the mathematical model, etc.)

Note that an ECFP repository is proposed. This repository process rules would indeed require fields of "complementary to" or "competing with" because the funding proposals need to be associated in this way to be processed by the Etherplan Consensus Engine.